homeassistant/components/xiaomi_aqara/__init__.py

```diff
@@ -110,6 +116,12 @@ def _fix_conf_defaults(config):
                 ),
                 vol.Optional(CONF_INTERFACE, default="any"): cv.string,
                 vol.Optional(CONF_DISCOVERY_RETRY, default=3): cv.positive_int,
+                vol.Optional(
+                    CONF_MOTION_SENSOR_NO_MOTION_TIMEOUT, default=120
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration is in the process of being converted to config flow and config entries. See:
https://github.com/home-assistant/core/pull/35595

Reply:
Thank you for the information. How is such a process handled normally. Should i wait until the linked breaking change is release and make my changes again?

Thanks in advance

Reply:
When that PR is merged into dev branch it's easier to start a new branch based off dev branch and apply changes in a new PR.

I suggest taking a discussion with the codeowners if this approach with the options is good.

Reply:
Okay, I will wait until the changed from the code owner are made. Further I will ask him about my changes. Should I close the merge request until then?

Reply:
Yes, please close while waiting. Thanks! :+1:

Reply:
Not sure if this is rude or the wrong procedure, but since #35595 is merged, can someone once again take a look at this?

Reply:
Hi there @Korni22,

This PR has been closed for months already. Could you please be so kind not to comment/revive closed/handled PRs? It notifies everybody but is not trackable at the same time.

Please create an issue instead. Thanks 👍